src/erlcloud_s3.erl

```diff
@@ -1014,6 +1015,47 @@ make_link(Expire_time, BucketName, Key, Config) ->
       true  -> lists:flatten([Config#aws_config.s3_scheme, Config#aws_config.s3_host, port_spec(Config), "/", BucketName, "/", Key])
   end.
 
+ -spec make_get_url_v4(integer(), string(), string()) -> iolist().
+
+make_get_url_v4(ExpireTime, BucketName, Key) ->
+    make_get_url_v4(ExpireTime, BucketName, Key, default_config()).
+  
+-spec make_get_url_v4(integer(), string(), string(), aws_config()) -> iolist().
+
+make_get_url_v4(ExpireTime, BucketName, Key, Config) ->
+  Region = aws_region_from_host(Config#aws_config.s3_host),
+  Host = "s3-" ++ Region ++ ".amazonaws.com",
```

Comment:
This is not correct because there is no `s3-us-east-1.amazonaws.com` [endpoint](http://docs.aws.amazon.com/general/latest/gr/rande.html#s3_region).  
`Config#aws_config.s3_host` should be used as a `Host`.